library/src/G4CMPConfigManager.cc

```diff
@@ -110,6 +113,12 @@ G4CMPConfigManager::G4CMPConfigManager()
     kaplanKeepPh(getenv("G4CMP_KAPLAN_KEEP")?atoi(getenv("G4CMP_KAPLAN_KEEP")):true),
     chargeCloud(getenv("G4CMP_CHARGE_CLOUD")?atoi(getenv("G4CMP_CHARGE_CLOUD")):0),
     recordMinE(getenv("G4CMP_RECORD_EMIN")?atoi(getenv("G4CMP_RECORD_EMIN")):true),
+    Empiricalklow(getenv("G4CMP_EMPIRICAL_KLOW")?strtod(getenv("G4CMP_EMPIRICAL_KLOW"),0):-1.),
+    Empiricalkhigh(getenv("G4CMP_EMPIRICAL_KHigh")?strtod(getenv("G4CMP_EMPIRICAL_KHigh"),0):-1.),
+    EmpiricalkFixed(getenv("G4CMP_EMPIRICAL_KFIXED")?strtod(getenv("G4CMP_EMPIRICAL_KFIXED"),0):-1.),
+    EmpiricalELow(getenv("G4CMP_EMPIRICAL_ELOW")?strtod(getenv("G4CMP_EMPIRICAL_ELOW"),0)*keV:-1),
+    EmpiricalEHigh(getenv("G4CMP_EMPIRICAL_EHIGH")?strtod(getenv("G4CMP_EMPIRICAL_EHIGH"),0)*keV:-1),
+    EmpiricalEnergyDependentK(getenv("G4CMP_EMPIRICAL_ENERGYDEPENDENTK")?strtod(getenv("G4CMP_EMPIRICAL_ENERGYDEPENDENTK"),0):-1),
```

Comment:
The use of strtod() here is not correct.  This variable is a boolean, and should be treated as such.  Look at other boolean variables in `G4CMPConfigManager` (e.g., `kaplanKeepPh`, `recordMinE`), and do the same thing.